### ISSUE TITLE
Select multiple files at once

### ISSUE THREAD

**Issue by mrandrey** [Natizyskunk/vscode-sftp#97]
**Is this a similar or duplicate feature request?**
- [ ] I don't know. I will go check it.
- [x] No.

**Is your feature request related to a problem? Please describe.**
I'd like to be able to select multiple files/folders at once on the remote server to download.

**Describe the solution you'd like**
Holding down Cmd/Ctrl/Shift while selecting to select multiple files at once, just like on regular explorer view. 
Also add "Download" context menu item on the remote (I added both folder & file download manually to package.json—it works fine).

**Describe alternatives you've considered**
Download files one by one.

**Does this project help you?**
- [x] Yes. SFTP IS AWESOME!

Thanks!

**Comment by Natizyskunk** [comment]
@mrandrey,

>>  added both folder & file download manually to package.json—it works fine

 This is great, can you do a pull request following the [Pull Request Guidelines](https://github.com/Natizyskunk/vscode-sftp/issues/80) please ?

**Comment by mrandrey** [comment]
@Natizyskunk,

Here's the code that I added to package.json to bring in Download File & Folder to the remote view:

After this:
```
				{
					"command": "sftp.revealInExplorer",
					"group": "2_files",
					"when": "view == remoteExplorer && viewItem != root"
				},
```
... add this:
```
				{
					"command": "sftp.download.file",
					"group": "2_files",
					"when": "view == remoteExplorer && viewItem != root && viewItem == file",
					"_comment": "!custom"
				},
				{
					"command": "sftp.download.folder",
					"group": "2_files",
					"alt": "sftp.forceDownload",
					"when": "sftp.enabled && viewItem != file",
					"_comment": "!custom - (using != file so that whole site can be downloaded)"
				},

```
I'm not sure how to do a pull request, but feel free to do it yourself if you think this is ready to be ported to the live version. 

And, my original question still stands:
How can I select multiple files at once on the remote view? For example, if I'd like to download all but one folder, right now I have to manually select each folder/file one by one.

Thanks!

**Comment by mrandrey** [comment]
That would be great. Thanks!

**Comment by NyaPPuu** [comment]
Related issues : #161

**Comment by Natizyskunk** [comment]
Merged in pull request #251 🙂
